src/shims/mapShim.ts

```diff
@@ -0,0 +1,212 @@
+/* @internal */
+namespace ts {
+    /* @internal */
+    // @ts-ignore

```

Comment:
?????????????????????????????????????????????? What's being ignored here?

Reply:
Unused type argument. It isn't considered used until it merges with the definition in core.ts.

Reply:
Oh, the extra `/* @internal */`s can go though, that was left over from me trying to figure out why the whole file kept being added to typescript.d.ts, which I fixed in #33694.

Reply:
You should include a comment on what the `//@ts-ignore` is supposed to be ignoring, then.

Reply:
Already done as of 057e184.